src/pages/Checkout/CheckoutPage.vue

```diff
@@ -502,6 +508,23 @@ export default {
 			);
 		}
 
+		// MARS-452 tip rate optimization experiment
+		if (!this.checkingOutAsGuest) {
```

Comment:
Because this is in the `created` hook, `this.checkingOutAsGuest` will always be `false`. It doesn't get set to `true` until the user clicks "Continue as guest". Maybe this should check for `this.myId` instead? I'm not sure if/how we can set this up before the user has logged in.